backend/spec/system/backoffice/open_calls_spec.rb

```diff
@@ -50,6 +50,17 @@
         end
       end
 
+      context "when searching by partial name" do
+        it "shows only found open calls" do
+          expect(page).to have_text(open_call.name[0..2])
+          open_calls.each { |o| expect(page).to have_text(o.name) }
+          fill_in :q_name_or_investor_account_name_or_country_name_or_department_name_or_municipality_name_i_cont, with: "Ultr"
```

Comment:
Here is still `_cont` which I am not 100% but it is not only prefix searching --> should also return results if text is found at middle of word...

Reply:
you're right, but I think in ransack there's no equivalent of "any word starts with" matcher, so I ended up ignoring this

Reply:
Hmm, checked https://activerecord-hackery.github.io/ransack/getting-started/search-matches/ and there are `*_start*` matchers, so maybe these would be usable in this case. It would just require to split full text sentence to words inside controller.

Reply:
Okay, this one will not work, because text at db is still un-split. Really the easiest to ignore it now :D. We could theoretically use `pg_search` instead of `ransacker` if we give up on translation fallback, but that is bigger refactoring  and probably overkill now